docs/resources-overview.md

```diff
@@ -33,13 +35,12 @@ To see only objects of a specific type, for example to see the webhook and
 controller deployments inside Knative Serving, you can run
 `kubectl -n knative-serving get deployments`.
 
-The Knative Serving controller creates Kubernetes and Istio resources when
-Knative Serving resources are created and updated. It will also create Build
-resources when provided in the Configuration spec. These sub-resources will be
-created in the same namespace as their parent Knative Serving resource, _not_
-the `knative-serving` namespace. For example, if you create a Knative Serivce in
-namespace 'foo' the corresponding Istio resources will also be in namespace
-'foo'.
+The Knative Serving controller creates Kubernetes resources when Knative Serving
+resources are created and updated. These sub-resources will be created in the
+same namespace as their parent Knative Serving resource, _not_ the
+`knative-serving` namespace. For example, if you create a Knative Serivce in
+namespace 'foo' the corresponding Deployment and ReplicaSet resources will also
+be in namespace 'foo'.
 
 All of these components are run as a non-root user (uid: 1337) and disallow
 privilege escalation.
```

Comment:
is this the right uid? 🤔

Reply:
I dont know, and I don't think we need to say the exact ID here anyway, so I'll just remove it